.github/actions/teamsfx-utils/utils.ts

```diff
@@ -9,8 +9,12 @@ export function setOutput(key: string, value: string) {
     fs.appendFileSync(output, `${key}=${value}${os.EOL}`);
 }
 
-export function getEmail(githubUser: string): string {
-    const accounts = fs.readJsonSync(path.join(__dirname, '../..', '.github', 'accounts.json'));
+export function getEmail(githubUser?: string): string {
+    if (!githubUser) {
+        return "";
+    }
+    const res = fs.readFileSync(path.join(__dirname, '../..', '.github', 'accounts.json'));
```

Comment:
better to path.join(__dirname, '..', '..', '.github', 'accounts.json')

Reply:
Fixed.